server/services/supportedPrisonsService.ts

```diff
@@ -1,25 +1,34 @@
 import { Prison, SystemToken } from '../@types/bapv'
-import prisons from '../constants/prisons'
+import PrisonRegisterApiClient from '../data/prisonRegisterApiClient'
 import VisitSchedulerApiClient from '../data/visitSchedulerApiClient'
 
 type VisitSchedulerApiClientBuilder = (token: string) => VisitSchedulerApiClient
+type PrisonRegisterApiClientBuilder = (token: string) => PrisonRegisterApiClient
+
+const A_DAY_IN_MS = 24 * 60 * 60 * 1000
```

Comment:
I've set it to cache for a day as I'm assuming it barely ever changes, but open to suggestions for shorter/longer time 🤷‍♂️ !